src/moves_1.c

```diff
@@ -1,7 +1,904 @@
 #include "global.h"
 #include "code_8092334.h"
+#include "memory.h"
 #include "moves.h"
 #include "pokemon.h"
+#include "subStruct_203B240.h"
+#include "code_809447C.h"
+#include "text_util.h"
+
+extern u8 gAvailablePokemonNames[];
+extern const char gUnknown_8109930[];
+extern struct MoveDataEntry *gMovesData;
+
+u8 sub_8093468(int param_1, struct Move* src_struct);
+u8 TryLinkMovesAfter(int param_1, struct Move* src_struct);
+u8 sub_80933D8(int param_1, void* src_struct);
+bool8 DoesMoveCharge(u16 move);
+
+void unk_GetLinkedSequences4(struct Move* moves, struct Move linkedSequences[4][4]);
+void unk_LinkedSequencesToMoves4(struct Move* moves, struct Move linkedSequences[4][4]);
+void unk_GetLinkedSequences8(struct Move* moves, struct Move linkedSequences[8][8]);
+void unk_LinkedSequencesToMoves8(struct Move* moves, struct Move linkedSequences[8][8]);
+
+void unk_GetLinkedSequences8_v2(struct Move*, struct Move[8][8]);
+void unk_LinkedSequencesToMoves8_v2(struct Move*, struct Move[8][8]);
+
+int unk_PrintMoveDescription(int, struct Move*, int, struct subStruct_203B240**);
+
+extern void sub_80073B8(u32);
+void sub_80928C0(u8 *buffer, struct Move *move, struct unkStruct_80928C0 *param_3);
+extern void xxx_format_and_draw(u32, u32, const u8 *, u32, u32);
+extern void unk_MovePrintData(struct Move*, int);  // print something
+extern void sub_80073E0(u32);
+extern u32 sub_8097DF0(char *, struct subStruct_203B240 **);
+
+extern void sub_80078A4(u32, u32, u32, u32, u32);
+extern u8 sub_8092B00(struct Move*);
+extern u32 sub_8092BC0(struct Move*);
+extern u32 gUnknown_202DE30;
+extern u8* gPtrTypeText;     // "Type"
+extern u8* gUnknown_810CF00; // "Range#=@.$m0 "
+
+
+int IsMoveSet(int index, struct Move* struct_ptr)
```

Comment:
Looks like many of these are `bool8` instead of `int`

Reply:
Along with changing the return type, `return 1` and `return 0` can be `return TRUE` and `return FALSE`.